docs/storage/prometheus.md

```diff
@@ -98,6 +98,8 @@ Metric name | Type | Description | Unit (where applicable) | option parameter |
 `container_threads_max` | Gauge | Maximum number of threads allowed inside the container | | process |
 `container_ulimits_soft` | Gauge | Soft ulimit values for the container root process. Unlimited if -1, except priority and nice | | process |
 
+* `*`This metrics can be misleading for pod with `hostNetwork: true` ([Details](https://github.com/google/cadvisor/issues/2615)).
```

Comment:
Could you use [footnote](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes) notation instead?